daemon-tests/src/lib.rs

```diff
@@ -812,10 +826,14 @@ impl Taker {
             .clone()
     }
 
-    pub fn latest_fees(&mut self) -> CompleteFee {
+    pub fn latest_complete_fee(&mut self) -> CompleteFee {
         self.first_cfd().aggregated().latest_fees()
     }
 
+    pub fn latest_fees(&mut self) -> SignedAmount {
+        self.first_cfd().accumulated_fees
+    }
+
```

Comment:
🔧 Could we document these methods? I have a hard time understanding why we need two different methods.

Reply:
I think the underlying problem is actually how `projection::Cfd` is modeled and how we expose values there. I will see to rename the convenience accessors in the tests for now and can add a comment if it helps. 
I thought about creating only one accessor for the test and then map around, but that's not better neither I feel 😅